The musl library until v1.1.20 provided the expected symbols by erlang.
Starting with 1.1.21 the __sigaction and __libc_sigaction are no longer present,
leading to strange errors like "dlsym: Resource temporarily unavailable" when erlang
is trying to setup the signal handlers.

musl 1.1.20-r3:
/ # readelf -a /lib/ld-musl-x86_64.so.1 | grep sigaction
   112: 0000000000046130   386 FUNC    GLOBAL DEFAULT    8 __libc_sigaction
   114: 00000000000462b2    38 FUNC    GLOBAL DEFAULT    8 __sigaction
   826: 00000000000462b2    38 FUNC    WEAK   DEFAULT    8 sigaction

musl 1.1.21-r0:
/ # readelf -a /lib/ld-musl-x86_64.so.1 | grep sigaction
   725: 000000000004453c    38 FUNC    WEAK   DEFAULT    8 sigaction

So this patch fixes the function to patch, from __libc_sigaction to sigaction.
Cause of this issue is:
https://git.musl-libc.org/cgit/musl/commit/src/signal/sigaction.c?id=9b14ad541068d4f7d0be9bcd1ff4c70090d868d3


--- old/erts/emulator/hipe/hipe_x86_signal.c
+++ new/erts/emulator/hipe/hipe_x86_signal.c
@@ -166,8 +166,8 @@
  * There are libc-internal calls to __libc_sigaction which install handlers, so we must
  * override __libc_sigaction rather than __sigaction.
  */
-#define NEXT_SIGACTION "__libc_sigaction"
-#define LIBC_SIGACTION __libc_sigaction
+#define NEXT_SIGACTION "sigaction"
+#define LIBC_SIGACTION _sigaction
 #define OVERRIDE_SIGACTION
 #ifndef _NSIG
 #define _NSIG NSIG
